appveyor.yml

```diff
@@ -50,12 +50,12 @@ environment:
   #   ADDPATH: C:\cygwin\bin;
   #   TOOLSET: gcc
   #   CXXFLAGS: cxxflags=-std=c++11
-  # - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-  #   ADDPATH: C:\mingw\bin;
-  #   TOOLSET: gcc
-  #   CXXFLAGS: cxxflags=-std=c++03
-  #   MAYFAIL: true
-  #   COMMENT: serialization and mingw compatibility issues
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+      ADDPATH: C:\mingw\bin;
+      TOOLSET: gcc
+      CXXFLAGS: cxxflags=-std=c++03
```

Comment:
The new way is `cxxstd=03` instead of using `cxxflags`, which allows testing `cxxstd=03,11` or even `cxxstd=03,11,14,1z` in one go instead of using multiple jobs (slow on Appveyor as they're serialized).

Reply:
Sorry, I just re-enabled an old stanza here.  The other ones should be using that.  If not, I'll adjust.

Reply:
All of the jobs at https://ci.appveyor.com/project/jeking3/uuid-gaamf/build/1.0.73-develop seem to be using `cxxflags`.

Reply:
I was thinking about Travis CI, I'll get this fixed; it's queued up behind https://github.com/boostorg/predef/pull/68 right now.